### ISSUE TITLE
Wordpress Plugin

### ISSUE THREAD

**Issue by xeagu** [livinginformation/PepeVote#12]
PepeVote is an example of federated, stake based voting. PepeVote is the testing ground for an idea that will allow decentralized groups of people to form federated communities and vote on community decisions. Implementing the functionality of PepeVote into a WordPress plugin would allow this voting system to be easily replicated into other communities.

**Comment by jdabs** [comment]
Yes, I believe WordPress' entrenched network effect is the next frontier for crypto. Many are already working on this, such as Steemit.

Are you able to lend a hand with implementation in WordPress?
